CSS Font Loading API fixes - FF104 (#19990)
mdn/content: files/en-us/web/api/fontface/ascentoverride/index.md

---
{{APIRef("CSS Font Loading API")}}

> The **`ascentOverride`** property of the {{domxref("FontFace")}} interface returns and sets the ascent metric for the font.
> This is the height above the baseline that CSS uses to lay out line boxes in an inline formatting context

Reviewer: ```suggestion
The **`ascentOverride`** property of the {{domxref("FontFace")}} interface returns and sets the ascent metric for the font,
the height above the baseline that CSS uses to lay out line boxes in an inline formatting context.
```

> This is equivalent to the {{cssxref("@font-face/ascent-override")}} descriptor of {{cssxref("@font-face")}}.

Reviewer: ```suggestion
This property is equivalent to the {{cssxref("@font-face/ascent-override")}} descriptor of {{cssxref("@font-face")}}.
```

## Value

A string. The possible values are `normal`, indicating that the metric used should be obtained from the font file, or a percentage.

> This take the same values as the {{cssxref("@font-face/ascent-override")}} descriptor.

Reviewer: This sounds weird. What about:
```suggestion
This property accepts the same values as the {{cssxref("@font-face/ascent-override")}} descriptor.
```

or something similar

Author: FWIW this should have been "This **takes** the same values" which is fine. I like your working more though.
